actionlib_msgs/GoalID goal_id
geometry_msgs/Pose pose
uint32 priority